classroom.rb

```diff
@@ -0,0 +1,19 @@
+class Classroom
+  attr_accessor :label, :student_group
+
```

Comment:
- [ ] Kindly make the `student_group` to have a read access.  This is because this class  should not have direct access to mutate the  `student_group`.
-[ ] This ensures that it remains unchanged after initialization. Using `attr_accessor` when you only need to read an attribute could lead to unintended changes in the object's state, which may introduce bugs or make the code harder to maintain.